Domain/Product.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace Simple_Inventory_Management_System.Domain
+{
+    class Product
+    {
+        private string name;
+        private double price;
+        private int quantity;
+        public string Name { get; set; }
+        public double Price { get; set; }
+        public int Quantity { get; set; }
+        public Product(string name, double price, int quantity)
```

Comment:
let's have a single line space between the different sections in our code to have a clean code and increase the readability of the code.

Same thing for all other parts of your code.
```suggestion
        public int Quantity { get; set; }
        
        public Product(string name, double price, int quantity)
```